src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_FrameworkElement.cs

```diff
@@ -401,6 +411,17 @@ public async Task When_Add_Element_Then_Load_Raised()
 			Assert.AreEqual(1, loadedCount, "loaded");
 		}
 
+		[TestMethod]
+		[RunsOnUIThread]
+		public async Task When_Add_Native_Child_To_ElementCollection()
+		{
+			var panel = new Grid();
+			var tbNativeTyped = (_View)new TextBlock();
```

Comment:
Adding a TextBlock in this case won't go through the case with an issue (it checks for `if (view is UIElement uiElement)`). 

Try adding an instance of this type instead :
https://github.com/unoplatform/uno/blob/4bbd0cef8efbb833fe3bc64cc7498a1dc88fc973/src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/NativeViewPage/NativeView.cs#L16

Reply:
@jeromelaban Actually the issue was when it _was_ a `UIElement` - without the `else`, it was trying to add the view twice (the second time wrapped in another view using `AdaptNative()` ).

Reply:
Ah, missed that! Alright then :)